docs/configuration/practical_example.md

```diff
@@ -0,0 +1,200 @@
+# Practical example of configuration
+
+The previous sections have explained what to do in configuration. This section explains you a practical example to let you understand the flow of configuration.
+
+This section demonstrates creation of an architecture file for `CARET_demos` on Jupyter Notebook. Defining inter-node and intra-node data path is also demonstrated. The following three steps are explained.
+
+1. Load an Architecture object on Jupyter Notebook
+2. Define an inter-node data path
+3. Define an intra-node data path
+
+The architecture file that will be created in the example is provided [here](https://raw.githubusercontent.com/tier4/CARET_demos/main/samples/end_to_end_sample/architecture.yaml).
+
+## Load an Architecture object on Jupyter Notebook
+
+Load an architecture object from recorded data, as explained section of [Load and save](./load_and_save.md)
+
+1. Launch Jupyter Notebook (Jupyter Lab)
+
+   ```bash
+   mkdir -p ~/ros2_ws/evaluate && cd ~/ros2_ws/evaluate
+
+   source ~/ros2_caret_ws/install/setup.bash
+   jupyter-lab
+   ```
+
+2. Generate an architecture file from recorded data as below
+
+   ```python
+   from caret_analyze import Architecture
+
+   # Read description of application's architecture from recorded data
+   #
+   arch = Architecture('lttng', './e2e_sample')
+
+   # Save description as an architecture file
+   arch.export('architecture.yaml')
+
+   # Check if the architecture file is created
+   ! readlink -f ./architecture.yaml
+   # /home/user/ros2_caret_ws/eval/architecture.yaml
+   ```
+
+## Define an inter-node data path
+
+Define an inter-node data path on the loaded architecture object as section of [Define inter-node data path](./inter_node_data_path.md) explained
+
+1. Load the yaml-based architecture file as below
+
+   ```python
+   from caret_analyze import Architecture, check_procedure
+   arch = Architecture('yaml', './architecture.yaml')
+   ```
+
+2. Choose source node and destination node in a path
+
+   `arch.search_paths` extract all candidates of the path
+
+   ```python
+   paths = arch.search_paths(
+   '/sensor_dummy_node', # source node
+   '/actuator_dummy_node') # destination node
+   ```
+
+   If a target application is large and complicated, `arch.search_paths` method may consume time more than 1 minute.
+   For decreasing consumed time, you can ignore nodes and topics and specify depth of search. Refer to [パスの探索方法](../configuration/inter_node_data_path.md) for more details.
```

Comment:
Japanese is left.
```suggestion
   For decreasing consumed time, you can ignore nodes and topics and specify depth of search. Refer to [Define inter-node data path](../configuration/inter_node_data_path.md) for more details.
```

Reply:
fixed at https://github.com/tier4/CARET_doc/pull/112/commits/ae9c0badff34553a6b8fa44310838446c9324d37